components/datasets/metadata/form/published-layer/card/styles.scss

```diff
@@ -0,0 +1,10 @@
+.c-published-layer-card {
+  border: 1px solid darkslategrey;
```

Comment:
`darkslategrey ` <- I'll assume this is a temporary workaround for styling.

Reply:
yep, just while we have a design prototype for this